src/Aspire.Dashboard/Otlp/Model/OtlpSpan.cs

```diff
@@ -101,7 +101,7 @@ public List<OtlpDisplayField> AllProperties()
 
         if (!string.IsNullOrEmpty(StatusMessage))
         {
-            props.Add(new OtlpDisplayField { DisplayName = "StatusMessage", Key = KnownTraceFields.StatusField, Value = Status.ToString() });
+            props.Add(new OtlpDisplayField { DisplayName = "StatusMessage", Key = KnownTraceFields.StatusMessageField, Value = StatusMessage });
```

Comment:
this was the bug?

Reply:
Yes. `status` and `status_message` had the same key.

Apparently very few frameworks include a status message with spans. It looks like Azure Functions does which is why the user hit this problem.

Reply:
This could still explode right or is this unique?

Reply:
It's unique. Key is `trace.status_message`. Even if there is a custom tag named `trace.status_message` the custom tag is prefixed with `unknown-`. So it ends up as `unknown-trace.status_message`.

(these keys are all internal)